### PR TITLE
Docker multi platform support

### LINKED ISSUE #136: arm64 docker images
Currently available docker images only support amd64. It would be great if arm64 images could be provided as well!

### PR DESCRIPTION
**Pull request by jonasengelmann**
closes #136

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -12,48 +12,39 @@ on:
       - 'main'
 
 jobs:
-  docker-build-ghcr:
+  docker-build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: docker/metadata-action@v5
```
Comment (bdon) [inline .github/workflows/publish.yml:19]: it seems like we lose the tagged versions on GHCR/Docker Hub by removing this, right? Is there an example somewhere we can follow that accomplishes this correctly?

Comment (jonasengelmann) [reply .github/workflows/publish.yml:19]: `docker/metadata-action@v5` is actually still there, it's [here](https://github.com/protomaps/go-pmtiles/blob/9554132801055d0462df53a145553a0e0d560724/.github/workflows/publish.yml#L24), but done for both images. I tried to follow [this](https://docs.docker.com/build/ci/github-actions/push-multi-registries/) example and also tested it on my fork. It pushed to both dockerhub and ghcr.

Comment (bdon) [reply .github/workflows/publish.yml:19]: Not sure why the actions are not running on this PR, but we'll see, I'll tag a patch release. Thanks!

Comment (bdon) [reply .github/workflows/publish.yml:19]: (Oh, because it was a PR to a PR)

**Pull request merged**